code/index.html

```diff
@@ -1,30 +1,164 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+  <!-- Using Meta Tags to Prevent Form Zooming -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0,user-scalable=0" />
+  <link rel="stylesheet" href="style.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Glory:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;1,100;1,200;1,300;1,400;1,500&display=swap"
+    rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+
+  <title>Sprinkle</title>
+</head>
+
+<body>
+  <body class="main-grid">
+    <header class="header">
+      <img class="logo" src="images/logo_sprinkle.png" alt="logo">
+      <ul class="menu">
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+        <li class="sign-in"><a href="">Sign in</a></li>
+      </ul>
+    </header>
+
+    <div class="hero-container">
+      <img src="images/donut_hero.jpg" alt="#">
+      <p class="centered-text">Let's sprinkle!</p>
+    </div>
+
+    <wrapper class="signup">
+
+      <!-- Form sign up -->
+      <div class="signup-box">
+        <form action="https://httpbin.org/anything" method="post" validate>
+          <h1 class="h1-signup">Sprinkle up</h1>
+          <p class="p-sprinkling">Start sprinkling now</p>
+          <div class="info-box">
+            <label for="fname">Name*</label><br>
+            <input type="text" id="fname" name="fname" placeholder="Name" name="name" required>
+
+            <br><label for="email">Email*</label><br>
+            <input type="email" id="email" name="email" placeholder="Email" name="email" required><br>
+
+            <label for="password">Password*</label><br>
+            <input type="password" id="password" name="password" placeholder="Password" name="password" required><br>
+            <br>
+          </div>
```

Comment:
I love br-tags but to get less code in the HTML-doc I think flexbox can be used here to structure the element in the same way. Display flexbox, flex-direction: column; and maybe padding-bottom to get more space for the input-"password"-element

Reply:
Thank you for input!